### PR TITLE
added automation testcases for editable table

### PR DESCRIPTION
**Pull request by 4manasa**

### CODE REVIEW

**Review by vishalshrm539 — APPROVED** [review]
Ran the EmbeddedData test and it looked good.

File: src/components/templates/SimpleTable/SimpleTableManual.tsx [hunk 1]
```diff
@@ -369,10 +370,10 @@ export default function SimpleTableManual(props) {
           </TableBody>
         </Table>
         {readOnlyMode && rowData && rowData.length === 0 && (
-          <div className='no-records'>No records found.</div>
+          <div className='no-records' id='no-records'>No records found.</div>
```
Comment (vishalshrm539) [inline src/components/templates/SimpleTable/SimpleTableManual.tsx:373]: Although I approved the PR but here, both id's are identical which should be avoided.
Probably you can use, something like this  'readonly-no-records', 'editable-no-records'.

File: src/components/templates/SimpleTable/SimpleTableManual.tsx [hunk 2]
```diff
@@ -334,6 +334,7 @@ export default function SimpleTableManual(props) {
                         <button
                           type='button'
                           className='psdk-utility-button'
+                          id='delete-button'
```
Comment (vishalshrm539) [inline src/components/templates/SimpleTable/SimpleTableManual.tsx:337]: Here also, I think you can assign different id's to these buttons that will allow us to identify `delete button` for a specific record/row. 
I've done something like that for Field Group.

File: src/components/templates/SimpleTable/SimpleTableManual.tsx (hunk 1 above)
Comment (4manasa) [reply src/components/templates/SimpleTable/SimpleTableManual.tsx:373]: yes id's should n't be identical but both div's will not render at once.

File: src/components/templates/SimpleTable/SimpleTableManual.tsx (hunk 2 above)
Comment (4manasa) [reply src/components/templates/SimpleTable/SimpleTableManual.tsx:337]: Oh okay, I was identifying the different row elements by nth value.

**Pull request merged**